packages/yoroi-ergo-connector/src/inject.js

```diff
@@ -329,6 +329,10 @@ function injectIntoPage(code) {
 }
 
 function shouldInject() {
+    if (true) {
+        // <TODO:CONNECTOR_202201>
+        return false;
+    }
```

Comment:
Why we are disabling it?

Reply:
For the 4.9 intermediate release. It will be enabled for 4.10